source/common/upstream/cluster_manager_impl.cc

```diff
@@ -353,6 +371,7 @@ ClusterManagerImpl::ClusterManagerImpl(
   init_helper_.onStaticLoadComplete();
 
   ads_mux_->start();
+  ENVOY_LOG(info, "Done with the first phase");
```

Comment:
is this log line intentional? might be confusing for users that aren't familiar with the initialization process

Reply:
Also a bit ambiguous what is meant by "first" here.. is it primary, etc..

Reply:
Removed.